composer.json

```diff
@@ -46,14 +45,14 @@
         "drupal/smart_trim": "^1.1",
         "drush/drush": "^10.0",
         "webflo/drupal-finder": "^1.2",
-        "webmozart/path-util": "^2.3",
-        "zaporylie/composer-drupal-optimizations": "^1.1"
```

Comment:
Does `zaporylie/composer-drupal-optimizations` not work with php 7.4 or something?

Reply:
I had it in my head that we didn't need it anymore, but that's only true if we are on composer 2.0. Reverting!